crates/frontend/src/utils.rs

```diff
@@ -338,21 +338,22 @@ pub fn parse_value(val: &str, config_type: ConfigValueType) -> Result<Value, Str
         ConfigValueType::Boolean => bool::from_str(val)
             .map(Value::Bool)
             .map_err(|_| "Invalid boolean".to_string()),
-        ConfigValueType::Integer => val
-            .parse::<i64>()
-            .map(|number| Value::Number(number.into()))
-            .map_err(|_| "Invalid integer".to_string()),
-        ConfigValueType::Number => val
-            .parse::<i64>()
-            .map(|number| Value::Number(number.into()))
-            .or_else(|_| {
-                f64::from_str(val)
-                    .ok()
-                    .and_then(|num| Number::from_f64(num).map(Value::Number))
-                    .ok_or_else(|| {
-                        "Invalid decimal format or precision issue".to_string()
-                    })
-            }),
+        ConfigValueType::Number | ConfigValueType::Integer => {
```

Comment:
We should treat these two differently, and parse them into their corresponding types. 
Integer - i32
Number - f32

Reply:
Open to opinions on this